src/components/Hero/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Hero.scss';
+
+const Hero = props => (
+  <div className="jumbotron jumbotron-fluid hero" style={{ backgroundImage: `url(${props.cover})` }}>
```

Comment:
Open Question: @adamstankiewicz the ticket mentioned that we would get Organization logo, Organization name and Degree name but there was no mention of the background cover image. This assumes it will also be provided by the api ?

Reply:
Yeah. good point. The wires I had at the time of writing the ticket only had a gray background so I didn't know it was going to be a cover image. But yeah, that would likely come from the API or it might be from the Netlify configuration. I'd say as long as you are able to customize the hero component's background image through props, we should be able to handle both cases once we have more information for that.